java-generic-mqtt/src/main/java/com/cumulocity/generic/mqtt/client/GenericMqttSubscriber.java

```diff
@@ -0,0 +1,18 @@
+package com.cumulocity.generic.mqtt.client;
+
+/**
+ * <code>GenericMqttSubscriber</code> is used subscribe to messages on a topic.
```

Comment:
nitpick: prefer 'Interface that must be implemented by an application that subscribes to a topic using {@link GenericMqttFactory#buildSubscriber(GenericMqttConnectionConfig)}.'

As before, no need to mention this class (what else would we be talking about).